roles/confluent.common/molecule/custom-vars/molecule.yml

```diff
@@ -0,0 +1,24 @@
+---
+driver:
+  name: docker
+platforms:
+  - name: custom
+    image: ../../../confluent.control_center/molecule/Dockerfile.j2
```

Comment:
This template seems to be used on different roles. Couldn't be placed upwards (e.g. `roles/molecule`) to avoid dependency with this role? or maybe moving it to `confluent.common`?

Reply:
Ya ive been thinking a lot about where to put the shared resources... putting it in confluent.common would not change much because the other molecule.ymls would then just have 
```
    image: ../../../confluent.common/molecule/Dockerfile.j2
```
and molecule isnt technically a role... so i dont think there should be a molecule dir under roles. 

My other idea is to have a directory test/ in the root of the repo and put all shared testing resources there, maybe use some symlinking as well. for the time being, im happy with the fact that there isn't any duplicating code

Reply:
agree. `confluent.variables_handlers` could be event better than `confluent.common` as a place for shared resources. Is just a bit odd to have references to `control_center` on tests, but this definitely just a nit :)